tools/PI/DevHome.PI/Pages/AppDetailsPage.xaml

```diff
@@ -25,7 +25,7 @@
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
 
-        <TextBlock x:Uid="AppDetailsTextBlock" FontSize="20" Margin="0,0,0,8"/>
+        <TextBlock x:Uid="AppDetailsTextBlock" FontSize="{StaticResource SubtitleTextBlockFontSize}" FontWeight="SemiBold" Margin="0,0,0,8"/>

```

Comment:
>SemiBold [](http://example.com/codeflow?start=113&length=8)

Should there a SubtitleTextBlockFontWeight static resource we should use?

Reply:
Yes I started doing that (there isn't one already defined), but all it would be is a re-definition of SemiBold, so a little overkill. Instead, at some point we'll consider a bigger reworking of the page/nav model and styles, per #3089 and #3063.